.github/workflows/iroha2-dev.yml

```diff
@@ -1,7 +1,9 @@
 name: I2::Dev::Publish
 
 on:
-  push:
+  workflow_run:
+    workflows: [I2::Base::Publish]
+    types: [completed]
     branches: [iroha2-dev]
```

Comment:
This is mis-implementation. The workflow was not triggered after this PR merge

Reply:
Problems:

- [ ] `workflow_run` event lacks `paths` filters
- [ ] Not sure if `completed` condition reacts to the preceding workflow skipped
- [ ] Use of [`workflow_run`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_run) needs to be committed to the default branch
	![image](https://user-images.githubusercontent.com/49983831/187644885-9a8b396e-0400-4b35-96f7-c6e82b5f5a55.png)

Reply:
Solution:
- Publish the base image not as a workflow but simply as an additional step to each workflow